Riot/Modules/SecretsRecovery/RecoverWithPassphrase/SecretsRecoveryWithPassphraseViewAction.swift

```diff
@@ -0,0 +1,24 @@
+/*
+ Copyright 2020 New Vector Ltd
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import Foundation
+
+/// SecretsRecoveryWithPassphraseViewController view actions exposed to view model
+enum SecretsRecoveryWithPassphraseViewAction {
+    case recover
+    case unknownPassphrase
```

Comment:
Could we use `useRecoveryKey` or `recoverByKey` here and everywhere in the code?
It looks like more the action we want to do. "Unknown passphrase" is just one of the reasons that can lead to the recover by key action.